library/src/main/java/com/github/amlcurran/showcaseview/ShowcaseView.java

```diff
@@ -261,6 +280,43 @@ public void overrideButtonClick(OnClickListener listener) {
         hasCustomClickListener = true;
     }
 
+    public void overrideButtonTwoClick(OnClickListener listener) {
+        if (shotStateStore.hasShot()) {
+            return;
+        }
+        if (mButtonTwo != null) {
+            if (listener != null) {
+                mButtonTwo.setOnClickListener(listener);
+            } else {
+                mButtonTwo.setOnClickListener(hideOnClickListener);
+            }
+        }
+        hasCustomClickListener = true;
+    }
+
+
+
+    public void overrideButtonsClick(OnClickListener listener1,OnClickListener listener2) {
```

Comment:
I'd prefer not to have this method, just having the two `overrideButtonTwoClick` and `overrideButtonClick` methods